flyway/src/test/groovy/io/micronaut/flyway/FlywayConfigurationOverloadedPropertiesSpec.groovy

```diff
@@ -0,0 +1,62 @@
+package io.micronaut.flyway
+
+import io.micronaut.context.exceptions.BeanInstantiationException
+import io.micronaut.inject.qualifiers.Qualifiers
+import org.flywaydb.core.Flyway
+import org.flywaydb.core.api.configuration.Configuration
+import org.flywaydb.core.api.pattern.ValidatePattern
+
+import javax.sql.DataSource
+
+class FlywayConfigurationOverloadedPropertiesSpec extends AbstractFlywaySpec {
+
+    void 'overloaded properties in FluentConfiguration can be set'() {
+        given:
+        run('spec.name': FlywayConfigurationPropertiesEnabledSpec.simpleName,
+                'flyway.datasources.movies.enabled': true,
+                'datasources.movies.url': 'jdbc:h2:mem:flyway2Db;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE',
+                'datasources.movies.username': DS_USERNAME,
+                'datasources.movies.password': DS_PASSWORD,
+                'datasources.movies.driverClassName': DS_DRIVER,
+                'flyway.datasources.movies.ignoreMigrationPatterns': '*:*',
+                'flyway.datasources.movies.locations': 'classpath:db/migration,classpath:othermigrations',
+                'flyway.datasources.movies.encoding': 'utf-8',
+                'flyway.datasources.movies.target': '1',
+                'flyway.datasources.movies.baseLineVersion': '1'
```

Comment:
```suggestion
                'flyway.datasources.movies.baseline-version': '2'
```

This wasn't setting the baselineVersion (due to incorrect capitalization in `baseLine`)

The default is `1`, so this wasn't showing up (hence the change to `2`)